### ISSUE TITLE
big int typing error

### ISSUE THREAD

**Issue by hamuk** [Urigo/graphql-scalars#45]
Hello!

I'm getting an error when including this module in my TypeScript project on build. The error is:
```
node_modules/graphql-scalars/dist/esnext/BigInt.d.ts:1:25 - error TS7016: Could not find a declaration file for module 'graphql-bigint'. '/home/jordan/source/phoenix-api/node_modules/graphql-bigint/index.js' implicitly has an 'any' type.
  Try `npm install @types/graphql-bigint` if it exists or add a new declaration (.d.ts) file containing `declare module 'graphql-bigint';`

1 import * as BigInt from 'graphql-bigint';
                          ~~~~~~~~~~~~~~~~


Found 1 error.
```
I'm using the newest version (0.4.6).

Cheers

**Comment by lookfirst** [comment]
Here is how to shut typescript up...

![image](https://user-images.githubusercontent.com/85355/62210177-b64dec00-b3c5-11e9-8537-e9ff39fcd1e8.png)

Add to tsconfig:"compilerOptions"...

```
    "typeRoots": [
      "types"
    ]
```

index.d.ts: `declare module 'graphql-bigint';`

**Comment by ardatan** [comment]
Fixed in 0.6.7!
